app/src/main/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -1256,14 +1256,12 @@ private void resumeWalletDownload() {
     view.getLifecycleEvent()
         .filter(lifecycleEvent -> lifecycleEvent == View.LifecycleEvent.CREATE)
         .flatMap(create -> view.resumePromotionDownload()
-            .flatMap(walletApp -> permissionManager.requestDownloadAccess(permissionService)
-                .flatMap(success -> permissionManager.requestExternalStoragePermission(
-                    permissionService))
-                .flatMapCompletable(
-                    __ -> appViewManager.resumeDownload(walletApp.getMd5sum(), walletApp.getId(),
-                        walletApp.getDownloadModel()
+            .flatMap(
+                walletApp -> permissionManager.requestExternalStoragePermission(permissionService)
```

Comment:
Same question of above related with the removal of download permission, but keeping the external storage